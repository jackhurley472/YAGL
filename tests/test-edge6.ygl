Graph g_all;
Graph g_add;

Node james("James");
Node jack("Jack");
Node adam("Adam");
Node tank("Tank");
Node random_person("Rando");

g_all: + james + jack -> james + adam ->2 james ->3 jack + tank ->4 adam ->5 jack 
       + random_person ->100 tank ->101 random_person - random_person;

printGraph(g_all);
/* Expectation before running:
	Nodes in graph: james (James), jack (Jack), adam (Adam), tank (Tank)
	Edges: Jack to James, Adam to James, James to Jack, Tank to Adam,
	       Adam to Jack.
*/

g_add + james;
g_add + jack;
g_add: + tank + adam;
printGraph(g_add);
/* Expectation before running:
	Nodes in graph: james (James), jack (Jack), adam (Adam), tank (Tank)
	No edges.
*/
