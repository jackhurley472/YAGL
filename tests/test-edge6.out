============== Graph Print ===============
	Stats: 	8	4	5

Nodes:
Node (0) : James --- Node (1) : Jack --- Node (2) : Adam
Node (3) : Tank

Edges:
[(1) : Jack] ---(1)---> [(0) : James]
[(2) : Adam] ---(2)---> [(0) : James]
[(2) : Adam] ---(5)---> [(1) : Jack]
[(0) : James] ---(3)---> [(1) : Jack]
[(3) : Tank] ---(4)---> [(2) : Adam]
============ End Graph Print =============
============== Graph Print ===============
	Stats: 	4	4	0

Nodes:
Node (0) : James --- Node (1) : Jack --- Node (3) : Tank
Node (2) : Adam

Edges:
There aren't any edges in this graph.
============ End Graph Print =============
